Makefile

```diff
@@ -3,7 +3,9 @@ format:
 	gradle ktlintFormat -p android
 
 lint:
-	Pods/SwiftLint/swiftlint
+	ios/Pods/SwiftLint/swiftlint --path ios
+	android/gradlew ktlint -p android
 
-init:
-	 git submodule update --force --recursive --remote
+deps:
+	rm -rf android/libraries; git submodule update --init --force --recursive
+	git submodule foreach 'git checkout master && git pull || :'
```

Comment:
Can you make this line `git submodule foreach 'git checkout $$sha1 && git pull || :'`

Reply:
Optionally, you can add a `depsUpdate` target that does checkout master